src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -752,11 +749,12 @@ public function matches_breadcrumbs( $breadcrumbs ) {
 	 *       this returns false for self-closing elements in the
 	 *       SVG and MathML namespace.
 	 *
-	 * @param  ?WP_HTML_Token $node Node to examine instead of current node, if provided.
-	 * @return bool Whether to expect a closer for the currently-matched node,
-	 *              or `null` if not matched on any token.
+	 * @param WP_HTML_Token $node Node to examine instead of current node, if provided.
+	 *
```

Comment:
my IDE auto-adds this newline here, but I thought WPCS requires it be absent

Reply:
I think the rules are more relaxed, although I also thought there were strict alignment rules that I'm not seeing either. I just try to keep it as readable as possible and sometimes will add space especially before the `@return` tag if it seems to help.

🤷 This change was covered after merging trunk so it doesn't matter anyways.